.github/dependabot.yaml

```diff
@@ -31,10 +31,15 @@ updates:
     commit-message:
       prefix: "deps(go-tools)"
 
-  ## Git Submodules
+  ## Subiquity dependencies
   - package-ecosystem: gitsubmodule
     directory: /
     schedule:
         interval: "daily"
         time: "02:00"
     target-branch: "main"
+    commit-message:
+      prefix: "deps(subiquity)"
+    allow:
+      - dependency-name: packages/subiquity_client
```

Comment:
Not sure how `dependency-name` works here, but I believe it should either be "subiquity" (the submodule's name) or "packages/subiquity_client/subquity" (the path to the submodule)

Reply:
I based it off of one of the [commits](https://github.com/canonical/ubuntu-desktop-provision/commit/8da5726a211c3d7dfee05d1ba1ae8f925064a7bf) from dependabot in the past?

Reply:
That's from before #181 though - that submodule doesn't exist anymore. `packages/subiquity_client` is a proper directory now, but it contains subiquity as a submodule in `packages/subiquity_client/subiquity`.

Reply:
Ah apologies! I am following now. Perfect, make that change :smile:

Reply:
No worries, the naming and submodule structure can be *slightly* confusing :D